test/integration/simple_operations_test.lua

```diff
@@ -391,3 +391,233 @@ pgroup:add('test_upsert', function(g)
     t.assert_equals(err, nil)
     t.assert_equals(result.rows, {{67, 1143, 'Mikhail Saltykov-Shchedrin', 63}})
 end)
+
+pgroup:add('test_get_partial_result', function(g)
+    -- insert_object
+    local result, err = g.cluster.main_server.net_box:call(
+            'crud.insert_object', {'customers', {id = 1, name = 'Elizabeth', age = 24}})
+
+    t.assert_equals(err, nil)
+    t.assert_equals(result.metadata, {
+        {name = 'id', type = 'unsigned'},
+        {name = 'bucket_id', type = 'unsigned'},
+        {name = 'name', type = 'string'},
+        {name = 'age', type = 'number'},
+    })
+    local objects = crud.unflatten_rows(result.rows, result.metadata)
+    t.assert_equals(objects, {{id = 1, name = 'Elizabeth', age = 24, bucket_id = 477}})
```

Comment:
Why 477?

Reply:
```
tarantool> vshard.router.bucket_id_strcrc32(1)
---
- 477
...
```

Reply:
Okay, but maybe should do it like this?

```lua
t.assert_equals(objects, {{id = 1, name = 'Elizabeth', age = 24, bucket_id = vshard.router.bucket_id_strcrc32(1)}}) 
```

or maybe leave a comment on how `477` is formed